elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -220,12 +228,11 @@ trait QueryDsl extends DslCommons with SortDsl {
     override def toJson(version: EsVersion): Map[String, Any] = Map(_lte -> value)
   }
 
-  case class WildcardQuery(key: String, value: String, boost: Option[Float] = None) extends Query {
+  case class WildcardQuery(key: String, value: String, boost: Option[Float] = None) extends Query with ValueBoost {
```

Comment:
Just to be sure: no `override val value/val value` is needed?

Reply:
No, Scala is smart enough to figure this out.